### PR TITLE
Replace ambient sounds from unknown sources with CC0 licensed ones

### PR DESCRIPTION
**Pull request by GraionDilach**
Samples:
rain https://www.freesound.org/people/vibe_crc/sounds/50058/
thunder1 https://www.freesound.org/people/netaj/sounds/193170/
thunder2 thunder-ambient
https://www.freesound.org/people/hifijohn/sounds/242735/
thunder3 https://www.freesound.org/people/sarson/sounds/195522/
thunder4 https://www.freesound.org/people/FenrirFangs/sounds/234737/

Solves #8841.

### CODE REVIEW

File: mods/ra/maps/fort-lonestar/fort-lonestar.lua [hunk 1]
```diff
@@ -152,9 +152,12 @@ Tick = function()
 		local delay = Utils.RandomInteger(1, 10)
 		Effect.Flash("LightningStrike", delay)
 		Trigger.AfterDelay(delay, function()
-			Media.PlaySound("thunder.aud")
+			Media.PlaySound(Utils.Random({"thunder1.aud", "thunder2.aud", "thunder3.aud", "thunder4.aud"}))
```
Comment (Phrohdoh) [inline mods/ra/maps/fort-lonestar/fort-lonestar.lua]: Can you just get a random integer and format it into `thunder{0}.aud`?

Comment (GraionDilach) [inline mods/ra/maps/fort-lonestar/fort-lonestar.lua]: Gave it a try, string.format didn't worked in Lua.

**Comment by GraionDilach** [comment]
Reverted my accidental chmod too.

**Comment by obrakmann** [comment]
Nice. The different thunder sounds made it even better. It's all CC0-licenced, too. :+1:

File: mods/ra/maps/fort-lonestar/fort-lonestar.lua (hunk 1 above)
Comment (abcdefg30) [inline mods/ra/maps/fort-lonestar/fort-lonestar.lua]: You can use
```
Media.PlaySound("thunder" .. Utils.RandomInteger(1,5) .. ".aud")

Comment (GraionDilach) [inline mods/ra/maps/fort-lonestar/fort-lonestar.lua]: Ah, didn't know it should be worded like that. Thanks, done.

**Comment by GraionDilach** [comment]
Couldn't resist adding https://www.freesound.org/people/Xythe/sounds/37299/ as thunder5 due to being a bit reminiscent to the famous Castle Thunders along with the fixup, sorry.

**Comment by abcdefg30** [comment]
Looks good to me now. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/a4adabed94666ae819ab17bdc9aa378e220b9ae7...90960ac00abf96613532a676162573c1583c2a37)

**Pull request merged**